src/lib/style-properties.html

```diff
@@ -158,12 +158,15 @@
             // case (2) variable
             var self = this;
             var fn = function(prefix, value, fallback, suffix) {
+              if (!value) {
+                return prefix + suffix;
+              }
               var propertyValue = self.valueForProperty(props[value], props);
               // if value is "initial", then the variable should be treated as unset
               if (!propertyValue || propertyValue === 'initial') {
                 // fallback may be --a or var(--a) or literal
                 propertyValue = self.valueForProperty(props[fallback] || fallback, props) ||
-                fallback;
+                fallback || 'unset';
```

Comment:
That would require knowing which properties are `inherit` and `initial` by default, including all shorthands.
Should we bother?